css/main.css

```diff
@@ -1,3 +1,58 @@
 /* Add your custom CSS here */
+*{
+    margin: 0px;
+}
+body {
+    font-family: "https://fonts.google.com/specimen/Quicksand";
+    color: rgb(0,0,0); 
+}
+
+.img-prof {
+    border-radius: 40%;
+}
+a {
+    padding:5px;
+    ;
+}
+
+.flex-nav{
+    display: flex;
+    justify-content: space-between;
+    padding: 5vh;
+}
+h1 h2 h3 h4 h5 h6 {
+    font-family: "https://fonts.google.com/specimen/Poppins";
```

Comment:
Again here you need to say `font-family: 'Poppins', sans-serif;` not include the link. The link to the asset should be inserted in the `head` of your HTML. 
See the right-hand panel [here for instructions](https://fonts.google.com/specimen/Poppins?query=poppins&selection.family=Poppins:wght@300&sidebar.open=true)